modules/transactions.js

```diff
@@ -102,6 +102,8 @@ __private.list = function(filter, cb) {
 	const params = {};
 	const where = [];
 	const allowedFieldsMap = {
+		senderIdOrRecipientId:
+			'"t_senderId" IN (${senderId:csv}) OR "t_recipientId" IN (${recipientId:csv})',
```

Comment:
`(${senderId:csv})` and `(${recipientId:csv})` should be `(${senderIdOrRecipientId:csv})`

Reply:
haha yes, just found it out myself ^^